_episodes/04-function-style.md

```diff
@@ -1,15 +1,20 @@
 ---
 title: "Python Coding Style"
+
```

Comment:
Again, not sure about adding this blank line since this is templated material. Have you rendered it locally to see if the template still works?

Reply:
I don't know how to do that. Is this a Jekyl thing?

Reply:
@eirrgang yes it is, you need to install [Jekyll](https://jekyllrb.com/docs/) and type [`jekyll serve`](https://stackoverflow.com/questions/51157446/whats-the-difference-between-bundle-exec-jekyll-serve-and-jekyll-serve) command in the main directory. Hope that works for you.